scenes/game/at-bat-manager.lua

```diff
@@ -0,0 +1,75 @@
+--------------------------------------------------------------------
+-- AtBatManager is a state machine that manages an individual at bat
+-- between a pitcher and a batter, including calculating the result of
+-- of an at bat
```

Comment:
I'm guessing "at bat" refers to the entire sequence of pitches & bat attempts between the pitcher and a batter?

Reply:
yah it'll be the full matchup including balls and strikes i think